server/examples/go/hello/builtin_hello/helloapp.go

```diff
@@ -95,7 +95,7 @@ func (h *helloapp) OneWay(call *apps.Call) error {
 }
 
 func (h *helloapp) Install(c *apps.Call) *apps.CallResponse {
-	if c.Type != apps.CallTypeSubmit {
+	if c.Type != apps.CallTypeSubmit && c.Type != "" {
```

Comment:
This is a call from the proxy. Can we assume this will be well formed (and therefore will never receive a empty string type)?

Reply:
I think we should get rid of all empty string checks, and instead return an error when there is an unexpected call type (like an empty one).

I had this check in here to make sure all the different pieces would work on everyone's computer (webapp inconsistency due to not being merged yet or something). But we should plan for the future and not really factor that in.